library/MedEx/API.php

```diff
@@ -2058,7 +2069,8 @@ public function login() {
         $this->curl->setData(array(
```

Comment:
Did not have to do anything to my standard windows set for my test box or the user that is getting the code